.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
If you set text=auto for all files, you need to ensure that binary files (like images) are excluded. Otherwise, git might detect a .jpg as a text file and try to normalize the line endings. It would be better to limit the normalization to .java files only.

See https://www.git-scm.com/docs/gitattributes#_effects